Crashlytics/Protogen/nanopb/crashlytics.nanopb.c

```diff
@@ -14,6 +14,22 @@
  * limitations under the License.
  */
 
```

Comment:
Something seems strange here that the copyright was added but the Generated version string was not updated.  Here and in subsequent three files.